### DISCUSSION TITLE
how to change the background color of window by  the   igl::opengl::glfw::Viewer

### DISCUSSION THREAD

**Discussion by jqdsn** [libigl/libigl#2129] [Q&A]
I want to change the background  color of window to white,does  the igl::opengl::glfw::Viewer of class  can do that ? 

![%(1K ~@24 24$QAOTXJ~ }L](https://user-images.githubusercontent.com/17001259/207286936-4799c3b4-dbc6-4ff4-adce-1ef80578b79a.png)

**Comment by BruegelN** [comment]
Hi @jqdsn 

`ViewerCore` has the property `background_color`: https://github.com/libigl/libigl/blob/87a550af22fc4af210af40f1d61d81594bbaf546/include/igl/opengl/ViewerCore.h#L137
In Tutorial 703 there is an example on how to set the background color to white: https://github.com/libigl/libigl/blob/87a550af22fc4af210af40f1d61d81594bbaf546/tutorial/703_Decimation/main.cpp#L124

**Reply by jqdsn** [reply]
I've set it up,thank you very much!
